content/blog/2021-07-27-july-21-dvc-community-gems.md

```diff
@@ -0,0 +1,132 @@
+
+You can achieve this by passing the `--idle-timeout=0` option to `cml-runner` in
+order to disable the timeout.
+
+Thanks @achbogga!
+
+### [Q: How can I change the default VPC to a different one with `cml-cloudrunner` for AWS?](https://discord.com/channels/485586884165107732/728693131557732403/857940793616498738)
+
+Great gem from @krish98409!
+
+You could setting the security group via `cloud-aws-security-group`. It will
+pick the VPC that manages that precise security group.
+
+We still don't provide a way of specifying VPCs other than the default one, but
+it's an issue that we're currently working on:
+https://github.com/iterative/terraform-provider-iterative/issues/107
+
+### [Q: Is it possible to rename and modify a file inside a directory tracked by DVC in one commit/change?](https://discord.com/channels/485586884165107732/485596304961962003/849589484517588992)
+
+If you modify the name and modify the file, you just need to run `dvc commit` or
+`dvc add` and then commit the change into Git.
+
+This was a good question for everyone. Thanks @snowpong!
+
+### [Q: How can I list the experiments I've queued?](https://discord.com/channels/485586884165107732/485596304961962003/856882434138570753)
+
+This is a great question to help us all understand something so thanks
+@adwivedi.
+
+To look at your queued experiments, run `dvc exp show`. All of the queued
+experiments will be marked with an asterisk `*`.
+
+_Queued experiments are not shown with the `dvc exp list` command at the
+moment._
+
+### [Q: I have two machines and a central repo on Azure. With my second machine, I want to pull the dataset from the first machine. How can I pull the data with DVC?](https://discord.com/channels/485586884165107732/485596304961962003/859034882297823233)
+
+Make sure that you have configured a DVC remote and run `dvc push` from your
+first machine. You should be able to find the files on the remote storage where
+you pushed them to after running that command. Then you can run `dvc pull` on
+your second machine and this should give you the dataset you pushed from the
+first machine.
+
+You will run into some issues if your remote isn't configured properly on the
+second machine. Check your `.dvc/config` file for the second machine to make
+sure there aren't any errors. It could be something as simple as a connection
+string without the necessary quotation marks!
+
+Thanks so much for this question @raharth!
+
+### [Q: `dvc push` says, "Everything is up to date." However, I modified my dataset and this is confirmed with `dvc status`, where it lists a "modified" entry on the changed outs. How can I force a push of my changes?](https://discord.com/channels/485586884165107732/485596304961962003/857931383476977695)
+
+You need to run `dvc add` to create a `.dvc` file to track the new data. Then
+run `dvc commit` to commit your changes to the cache.
```

Comment:
```suggestion
You need to run `dvc commit` to commit your changes to the cache.
```

Again, it seems like this question is about data that's already tracked, so IMO `dvc add` is a bit misleading in the answer.